build.gradle

```diff
@@ -26,6 +28,7 @@ dependencies {
     testImplementation group: 'io.cucumber', name: 'cucumber-picocontainer', version: '6.9.0'
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.6.0'
     testImplementation group: 'org.testng', name: 'testng', version: '7.3.0'
+    testImplementation 'io.qameta.allure:allure-cucumber6-jvm:2.13.6'
```

Comment:
Please consider changing this dependency declaration to:
`testImplementation group: 'io.qameta.allure', name: 'allure-cucumber6-jvm', version: '2.13.6'`
In order to standarize the dependencies declaration format.